dspace-xmlui/src/main/java/org/dspace/app/xmlui/opensearch/AbstractOpenSearchGenerator.java

```diff
@@ -122,8 +122,8 @@ public SourceValidity getValidity()
     {
         if (this.validity == null)
         {
```

Comment:
Here is the problem. the Constructor of ExpiresValidity(expiry) expects a delta. But the current code passes the delta **+ the time since Thu, 01 Jan 1970 00:00:00 GMT**